src/olympia/devhub/templates/devhub/base.html

```diff
@@ -35,3 +35,17 @@
 {% block footer_extras %}
   <img class="footerlogo" src="{{ static('img/developers/hub-logo-footer.png') }}" alt="">
 {% endblock %}
+
+{% block site_notice %}
+  {% if user.is_authenticated and user.is_survey_eligible(survey_id=amo.DEV_EXP_SURVEY_ALCHEMER_ID) and switch_is_active('enable_dev_experience_survey') %}
+    <div id="dev-survey-banner" response-url="{{ url('devhub.survey_response', survey_id=amo.DEV_EXP_SURVEY_ALCHEMER_ID) }}">
+      <div>
+        Help us make things better!
+        <a target="_blank" href="{{amo.SURVEY_LINK[amo.DEV_EXP_SURVEY_ALCHEMER_ID]}}" class="survey-link">Take our quick survey</a> 🦊
+      </div>
+      <a href="#" class="survey-dismiss">
+        ✕
+      </a>
+    </div>
+  {% endif %}
```

Comment:
You need a `{{ block.super }}` else you won't see the site notice/read only message if it's active. In fact, I'm surprised this is not breaking any tests ? Maybe we should have one checking that this (site notice/ready only message) is displayed correctly in devhub when active!